src/main.ts

```diff
@@ -82,13 +84,39 @@ function fileWatch(configs: webpack.Configuration[], args: any): Promise<void> {
 			}
 			if (stats) {
 				const runningMessage = args.serve ? `Listening on port ${args.port}` : 'watching...';
-				logger(stats.toJson(), configs, runningMessage);
+				logger(stats.toJson(), configs, args.target === 'lib', runningMessage);
 			}
 			resolve();
 		});
 	});
 }
 
+let libEntryPath: string;
+function getLibEntryPath(elements: Array<{ name: string; path: string }>) {
+	if (libEntryPath) {
+		return libEntryPath;
+	}
+
+	const basePath = process.cwd();
+	libEntryPath = path.join(basePath, 'src', 'main.ts');
+	if (!fs.existsSync(libEntryPath)) {
+		const tmpDir = fs.mkdtempSync(os.tmpdir());
```

Comment:
Thanks for your work on this. It's going to be very useful!

Whilst trying to get this setup I've found this line doesn't work on linux as `os.tmpdir` [does not include the path separator](https://nodejs.org/api/fs.html#fs_fs_mkdtemp_prefix_options_callback). This means the build tries to make a folder in the root of the filesystem.

[util.ts in this repo](https://github.com/mwistrand/cli-build-widget/blob/35-library-target/src/util.ts#L51) already works around it with:

```fs.mkdtempSync(`${os.tmpdir()}${path.sep}`)```

I'm curious about on which OS this works for you?

Reply:
Thanks for catching this. It's been working for me on macOS and Windows 10/11, but I'll update this as well.